src/ClientContextInterface.php

```diff
@@ -147,7 +147,7 @@
  * @method $this mset(array $dictionary)
  * @method $this msetnx(array $dictionary)
  * @method $this psetex($key, $milliseconds, $value)
- * @method $this set($key, $value, $expireResolution = null, $expireTTL = null, $flag = null)
+ * @method $this set($key, $value, bool $getOld = false, $expireResolution = null, $expireTTL = null, $flag = null)
```

Comment:
This will be a major breaking change. Do we need to break the argument order?

Reply:
@tillkruss No, we actually don't. The only reason I did like this is just to keep and order that Redis describes in docs, but I found that it can be in any order, so I just fix it to not being breaking change